tests: Shadowing the built-in function

y-bharath14 opened the pull request:
Shadowing the built-in function
Ex:
def set_seq_id(obj_type, router, **id <---**,  obj_name):
    ...
	...
	seq_id = int(seq_id) + int(id)  

    def __exit__(self, **type <---**, value, traceback):
        self.cleanup()

ton31337 reviewed (changes requested):
Please separate unrelated changes into the seprate commits, because this way is broken in terms of maintainability.

y-bharath14 commented:
> Please separate unrelated changes into the seprate commits, because this way is broken in terms of maintainability.

Hello Donatas Abraitis,

Addressed the review comment.
I will take up unrelated changes in the next PR.

Thanks & Regards,
Bharath
